### ISSUE TITLE
"maximum recursion depth exceeded" when trying to be obstinate and go to the bottom of things

### ISSUE THREAD

**Issue by jjst** [ana0/youarearock#1]
I admit it. I was obstinate. I didn't want to open my eyes. I wanted to see what it was like. Or rather not see, in a way, for the world seemed too dull and depressing.

So yes, I was obstinate, but I then decided to go straight to the bottom of things, because hey, I'm obstinate, but also lazy. But then this happened:

```
  File "/home/jjst/dev/youarearock/questionnode.py", line 177, in spasm
    self.spasm(standardscreen)
  File "/home/jjst/dev/youarearock/questionnode.py", line 177, in spasm
    self.spasm(standardscreen)
  File "/home/jjst/dev/youarearock/questionnode.py", line 177, in spasm
    self.spasm(standardscreen)
  File "/home/jjst/dev/youarearock/questionnode.py", line 177, in spasm
    self.spasm(standardscreen)
  File "/home/jjst/dev/youarearock/questionnode.py", line 177, in spasm
    self.spasm(standardscreen)
  File "/home/jjst/dev/youarearock/questionnode.py", line 177, in spasm
    self.spasm(standardscreen)
  File "/home/jjst/dev/youarearock/questionnode.py", line 177, in spasm
    self.spasm(standardscreen)
  File "/home/jjst/dev/youarearock/questionnode.py", line 177, in spasm
    self.spasm(standardscreen)
  File "/home/jjst/dev/youarearock/questionnode.py", line 177, in spasm
    self.spasm(standardscreen)
  File "/home/jjst/dev/youarearock/questionnode.py", line 175, in spasm
    standardscreen.addstr("     " + str(self.eternity))
RuntimeError: maximum recursion depth exceeded while getting the str of an object
```

TL;DR: The following series of actions `1 -> 2 -> 1 -> smash enter until next screen -> 1` leads to a maximum recursion depth exceeded.

**Comment by ana0** [comment]
Haha, hi Jeremie! Thanks for confessing to being obstinate :)

This is how the game ends.  You went to the very bottom of things and it was the maximum recursion depth.  

DON'T TELL ANYONE NO SPOILERS

**Comment by jjst** [comment]
Mind = :boom:

**Comment by pjf** [comment]
> DON'T TELL ANYONE NO SPOILERS

Too late (for me). :)

**Comment by ana0** [comment]
Haha!  Spoiler risk aside, I love this issue - it must remain!!

**Comment by makew0rld** [comment]
"It's not a bug, it's a feature!" - WONTFIX 😄
